### ISSUE TITLE
NEWSticker

### ISSUE THREAD

**Issue by lehr-ruetsche** [rpi-virtuell/reliwerk#208]
Da ich den Newsticker nicht bekomme unter meiner normalen Email-Adresse, habe ich versucht, ihn unter einem Alias zu abonnieren. Habt ihr jemals einen NEWSletter bei uns abboniert? Ich verstehe dieses Vorlage nicht. Normalerweise hakt man an: ich will diesen Newsletter abonnieren. Dann kriegt man eine Bestätigungsmail und gut ist.
Bei uns sieht das so aus, wie in der Anlage. Was heißt bitte SENT? Was sollen die ganzen Daten? Von meiner letzten Aktivität? Was heißt der Status "eingetragen". Das ich eingetragen bin? Das ich mich eintragen will? Ehrlich, ich verstehe es nicht. Ich wünschte mir, es wäre einfach einfach.
![newsletter](https://user-images.githubusercontent.com/17722241/34294886-d9bdee1c-e70a-11e7-8563-44cfbbceed45.png)

Man kann übrigens auch den Namen nicht einfach eintragen. Da muss man nochmals auf ein anderes Feld.

**Comment by staude** [comment]
Da wurde das falsche Formular eingebunden, wenn man nicht angemeldet ist lautet der inhalt der seite

Das Formular zur Abonnement-Verwaltung ist nur für Abonnenten einer Mailingliste verfügbar.

**Comment by lehr-ruetsche** [comment]
ja, das war mein lehr-ruetsche Formular. Da habe ich den Screenshot vom falschen Formular gemacht! Ich habe mich mit helenmarian angemeldet und wollte einen NEWSletter. Das war das Formular, das ich gekriegt habe:
![newsletter helenmarian](https://user-images.githubusercontent.com/17722241/34295202-4c8c4136-e70c-11e7-8c8c-5024719a4820.png)

Nicht wirklich viel besser, oder?

**Comment by staude** [comment]
Da wurde das falsche Formular eingebunden, wenn man **nicht angemeldet** ist lautet der inhalt der seite

> Das Formular zur Abonnement-Verwaltung ist nur für Abonnenten einer Mailingliste verfügbar.

**Comment by lehr-ruetsche** [comment]
Wenn ich mich NICHT anmelde, bekomme ich das:
![newsletter3](https://user-images.githubusercontent.com/17722241/34295475-e12ffd2c-e70d-11e7-8fe5-f05582ea2080.png)

Das ist auch nicht ganz klar. Hier müsste stehen: Wenn Sie den Newsletter abonnieren wollen, müssen Sie sich entweder anmelden oder registieren lassen.

**Comment by staude** [comment]
Werden meine Kommentare hier überhaupt gelesen?

**Comment by lehr-ruetsche** [comment]
Ich verstehe sie nicht. Du sagst, es wäre ein falsches Formular eingebunden? Wenn ich angemeldet bin? Also habe ich mich nicht angemeldet. Aber egal wie, irgendwie ist es nich so einfach verständlich, wie es sein könnte.

**Comment by joerglohrer** [comment]
Da war im Footer ein falscher Link zu einem Formular der Vorversion verlinkt. Ich habe das aktualisiert und auch das richtige Formular eingebunden. Jetzt sollte es unter [https://news.rpi-virtuell.de/newsletter-anmeldung/](https://news.rpi-virtuell.de/newsletter-anmeldung/) so aussehen:
![image](https://user-images.githubusercontent.com/1091954/34295755-4b053aae-e70f-11e7-8cf9-486c6df1e536.png)

**Comment by joerglohrer** [comment]
Hab's getestet und bekomme auch korrekt eine Bestätigungsmail:
![image](https://user-images.githubusercontent.com/1091954/34295797-880e7cc6-e70f-11e7-83d3-223ab01e23a4.png)

**Comment by joerglohrer** [comment]
läuft's jetzt wie gewünscht @lehr-ruetsche ? dann könntest du diesen Issue schließen.

**Comment by lehr-ruetsche** [comment]
Ich habe mir den Newsletter unter meinem Alias bestellt und hoffe, dass es klappt. Unter meiner web.de Seite, wohin die Alias-Post geschickt wird, habe ich allerdings schon vier NEWSletter gefunden. Würde das System es merken, wenn eine Adresse bereits vergeben ist?

**Comment by lehr-ruetsche** [comment]
Was ist bei den persönlichen Einstellungen der Unterschied zwischen "Benutzername" und "Spitzname"?

**Comment by lehr-ruetsche** [comment]
Unter meiner normalen kabelbw-Adresse habe ich (genauso wie Bernd) die letzten NEWSletter nicht erhalten. Nun habe ich es nochmal probiert und mich aufs neue angemeldet. Mal sehen, was jetzt passiert. Der Anmeldevorgang war genauso einfach, wie ich es mir gewünscht hatte :-)

**Comment by joerglohrer** [comment]
läuft es jetzt und kann geschlossen werden,  @lehr-ruetsche ?
